src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-*",
```

Comment:
Yes, Begin/End (APM) will be replaced by Task Async in the final contract. We may end up providing the legacy APM apis in a different contract or an adapter that converts from TPL to APM.
/cc: @SidharthNabar @davidsh @josguil